### PR TITLE
Use 'index-url' instead of 'find-links' to install domain libraries during CI testing

### PR DESCRIPTION
**Pull request by NivekT**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #1091
* #1090

Differential Revision: [D44111515](https://our.internmc.facebook.com/intern/diff/D44111515)

### CODE REVIEW

**Review by ejguan — APPROVED** [review]
Thank you!

**Comment by NivekT** [comment]
@nivekt has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D44111515).

**Comment by NivekT** [comment]
@nivekt has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D44111515).

File: .github/workflows/domain_ci.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Install torch and torchvision from nightlies
         run: |
           pip install numpy networkx
-          pip install --pre torch torchvision -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
+          pip install --pre torch torchvision --index-url https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
```
Comment (ejguan) [inline .github/workflows/domain_ci.yml:35]: My bad, it should be
```suggestion
pip install --pre torch torchvision --index-url https://download.pytorch.org/whl/nightly/cpu
```

**Comment by NivekT** [comment]
@nivekt has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D44111515).

**Comment by NivekT** [comment]
@nivekt has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D44111515).

**Pull request closed without merging**